apps/src/lab2/views/components/PredictSolution.tsx

```diff
@@ -0,0 +1,63 @@
+import {useAppSelector} from '@cdo/apps/util/reduxHooks';
+import React from 'react';
+import {PredictQuestionType} from '@cdo/apps/lab2/levelEditors/types';
+import Typography from '@cdo/apps/componentLibrary/typography';
+import moduleStyles from './predict.module.scss';
+import commonI18n from '@cdo/locale';
+
+// Component that displays the solution to a predict question.
+// The backend will only send the solution if the user has permission to see it.
+const PredictSolution: React.FunctionComponent = () => {
+  const predictSettings = useAppSelector(
+    state => state.lab.levelProperties?.predictSettings
+  );
+
+  if (!predictSettings?.solution) {
+    return null;
+  }
+
+  const getFormattedSolution = () => {
```

Comment:
minor: might be worth putting this in a `useMemo` that only runs when `predictSettings` updates, but not necessary

Reply:
I don't think this is complicated enough to warrant memoization, lmk if that is incorrect!